content/Hardware/RSeries/RSeriesNetworkPortIDs.md

```diff
@@ -5,17 +5,222 @@ weight: 40
 
 {{< toc >}}
 
-The network ports on a TrueNAS R-Series system are identified differently based on what software is installed.
+TTrueNAS identifies the network ports on TrueNAS R-Series systems based on what software is installed.
 
 {{< include file="/_includes/NetworkPortIdentifications.md" type="page" >}}
 
 By default, a TrueNAS R-Series with CORE installed identifies its network ports with **ixl** followed by a number.
 
 SCALE identifies its network ports with **eno** followed by a number.
 
-The tables below list the default identification for R-Series systems and any add-on networking cards that have been qualified for use with an R-Series system.
+The tables within the expandable boxes below list the default identification for R-Series systems and any add-on networking cards that iXsystems has qualified for use with R-Series systems.
 
-## R10 / R20 / R20A / R40 / R50
+## R10
+
```

Comment:
Why are you using the code emphasis for the port values? if not in a or as command, using the ` ` tags is not appropriate emphasis. You could add a column to these tables for Port number, then the value in another column but still not with the command emphasis.